### ISSUE TITLE
a scrollbar appears using auto size + autoclose

### ISSUE THREAD

**Issue by limo49** [Flyer53/jsPanel4#106]
jsPanel.create({panelSize:"auto", content:"hello", autoclose:3000});

The panel should resize vertically including the progressbar without showing scrollbars.

**Comment by Flyer53** [comment]
@limo49 
Am improvement of this is currently in the making and will be in the v4.9.0-alpha.11 update.

Meanwhile you could either add option `contentOverflow` like
```javascript
jsPanel.create({
  panelSize:"auto",
  content:"hello",
  autoclose:3000,
  contentOverflow:"hidden"
});
```
or resize the panel after creation like
```javascript
jsPanel.create({
  panelSize:"auto",
  content:"hello",
  autoclose:3000
}).resize("auto");
```

**Comment by Flyer53** [comment]
@limo49 
I just uploaded [v4.9.0-alpha.11](https://alpha.jspanel.de/downloads/jspanel-4.9.0-alpha.11.zip) which should fix this issue ...

**Comment by Flyer53** [comment]
fixed with v4.9.0 release
